config/config.go

```diff
@@ -417,9 +430,13 @@ type Config struct {
 	BuildCacheS3AccessKeyID     string `config:"build-cache-s3-access-key-id"`
 	BuildCacheS3SecretAccessKey string `config:"build-cache-s3-secret-access-key"`
 
-	PayloadFilterExecutable string `config:"payload-filter-executable"`
-	ProgressType            string `config:"progress-type"`
-	Infra                   string `config:"infra"`
+	PayloadFilterExecutable     string `config:"payload-filter-executable"`
+	ProgressType                string `config:"progress-type"`
+	Infra                       string `config:"infra"`
+	StackdriverTraceAccountJSON string `config:"stackdriver-trace-account-json"`
+	StackdriverProjectID        string `config:"stackdriver-project-ID"`
```

Comment:
`config:"stackdriver-project-ID" should perhaps be `config:"stackdriver-project-id" (lowercase id)